lib/analytics.js

```diff
@@ -11,340 +11,14 @@
  * Please see the LICENSE included with this distribution for details.
  */
 'use strict';
+var analyticsEvents = [];
 
-var afs = require('./fs'),
-	auth = require('./auth'),
-	async = require('async'),
-	colors = require('colors'),
-	fork = require('child_process').fork, // eslint-disable-line security/detect-child-process
-	fs = require('fs-extra'),
-	getOSInfo = require('./environ').getOSInfo,
-	i18n = require('./i18n')(__dirname),
-	mix = require('./util').mix,
-	path = require('path'),
-	request = require('request'),
-	timestamp = require('./time').timestamp,
-	urlEncode = require('./net').urlEncode,
-	uuid = require('uuid'),
-	__ = i18n.__,
-	__n = i18n.__n,
-
-	analyticsEvents = [],
-	url = 'https://api.appcelerator.com/p/v1/app-track',
-	sessionTimeout = 60 * 60 * 1000; // 1 hour
-
-/**
- * An array of events that are queued up to be sent.
- */
 exports.events = analyticsEvents;
 
-/**
- * Adds an event to the array of events to be sent to the Appcelerator cloud.
- * @param {String} name - The name of the event
- * @param {*} data - The data payload
- * @param {String} type - The event type
- */
-exports.addEvent = function addEvent(name, data, type) {
-	analyticsEvents.push({
-		id: uuid.v4(),
-		type: type || 'app.feature',
-		name: name,
-		ts: timestamp(),
-		data: data
-	});
+exports.addEvent = function addEvent(_name, _data, _type) {
+	analyticsEvents.push({});
```

Comment:
This function could be a noop. I don't think we need to populate the events array. With that said, it's harmless.

Reply:
Ah right. I had to use it for the test file, but since that is gone I can remove it :-)